src/bpfilter/cgen/cgen.h

```diff
@@ -127,12 +127,22 @@ int bf_cgen_unload(struct bf_cgen *cgen);
 
 void bf_cgen_dump(const struct bf_cgen *cgen, prefix_t *prefix);
 
+/**
+ * @enum bf_counter_type
+ *
+ * Special counter types for @ref bf_cgen_get_counter.
```

Comment:
`@ref bf_cgen_get_counter.` -> `@ref bf_cgen_get_counter .` otherwise Doxygen assumes the symbol name is `bf_cgen_get_counter.`

Reply:
Done.